### PR TITLE
Remove ProjectGUIDs from project files and Project and Name attributes from P2Ps

### PR DESCRIPTION
**Pull request by ViktorHofer**
https://github.com/dotnet/project-system/issues/232

ProjectGUIDs and Name arguments aren't necessary anymore and can be removed. There was one Name attribute that didn't match the project file name but I doubt that should cause issues as the name isn't used directly.

Regexes used
```
[^\S\r\n]*<ProjectGuid>{.+}</ProjectGuid>\s*

```

```
<Project>{.+}</Project>
```

```
<Name>.+</Name>
```

cc @safern @ericstj

### CODE REVIEW

File: src/System.Runtime.Extensions/tests/CustomAttributesTestApp/CustomAttributesTestApp.csproj [hunk 1]
```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```
Comment (stephentoub) [inline src/System.Runtime.Extensions/tests/CustomAttributesTestApp/CustomAttributesTestApp.csproj]: Why is the PR adding .csproj files?

**Review by stephentoub — APPROVED** [review]
Other than my one question, LGTM.

File: src/System.Runtime.Extensions/tests/CustomAttributesTestApp/CustomAttributesTestApp.csproj (hunk 1 above)
Comment (ViktorHofer) [reply src/System.Runtime.Extensions/tests/CustomAttributesTestApp/CustomAttributesTestApp.csproj]: My environment was dirty, that file belongs to another branch. Just went over the diff, that was the only mistake. Thanks for pointing out.

**Review by safern — COMMENTED** [review]
LGTM

**Review by ericstj — COMMENTED** [review]
LGTM

**Pull request merged**